### PR TITLE
Implement PORT

### PR DESCRIPTION
**Pull request by nabijaczleweli**

### CODE REVIEW

File: ISA.md [hunk 1]
```diff
@@ -62,13 +62,20 @@ Bit Mask  | Name | Count | Description
...
 1111 10XX | STCK |     4 | Stack manipulation, see section below
 1111 110X |      |     2 | Reserved
 1111 1110 | CLRF |     1 | Clear the 'F' register, by setting it to `0000 0000`
 1111 1111 | HALT |     1 | Stop the CPU from doing any more execution 
 
+## PORT - I/O
+
+The PORT instruction in the form 1111 DAAA will perform I/O on the port specified in register A.
+
+The `D` bit specifies the direction - `1` for reading in from the port and `0` for writing out to the port.
```
Comment (thecoshman) [inline ISA.md]: Can we clearly state the mnemonics for these two, either `PORT IN` and `PORT OUT` or, and I think this is even better, `PIN` and `POUT`... pout pout

File: ISA.md [hunk 2]
```diff
@@ -62,13 +62,20 @@ Bit Mask  | Name | Count | Description
...
+1110 XXXX | PORT |    16 | Perform I/O, see section below
 1111 0AAA | COMP |     8 | Compare register S with register `AAA`, see section below
 1111 10XX | STCK |     4 | Stack manipulation, see section below
 1111 110X |      |     2 | Reserved
 1111 1110 | CLRF |     1 | Clear the 'F' register, by setting it to `0000 0000`
 1111 1111 | HALT |     1 | Stop the CPU from doing any more execution 
 
+## PORT - I/O
+
+The PORT instruction in the form 1111 DAAA will perform I/O on the port specified in register A.
```
Comment (thecoshman) [inline ISA.md]: > the form `1110 DAAA`

**Review by thecoshman — CHANGES_REQUESTED** [review]

File: ISA.md (hunk 1 above)
Comment (nabijaczleweli) [reply ISA.md]: I went with `PORT IN`/`PORT OUT` because this is not x86 and we don't need to repeat its mistakes. Fixed.

File: ISA.md (hunk 2 above)
Comment (nabijaczleweli) [reply ISA.md]: Fixed.

**Pull request merged**